includes/class-wp-job-manager-cache-helper.php

```diff
@@ -128,6 +129,105 @@ public static function clear_expired_transients() {
 			$wpdb->query( $wpdb->prepare( $sql, $wpdb->esc_like( '_transient_jm_' ) . '%', $wpdb->esc_like( '_transient_timeout_jm_' ) . '%', time() ) );
 		}
 	}
+
+	/**
+	 * Maybe remove pending count transients
+	 *
+	 * When a supported post type status is updated, check if any cached count transients
+	 * need to be removed, and remove the
+	 *
+	 * @since 1.6.2
+	 *
+	 * @param string  $new_status New post status.
+	 * @param string  $old_status Old post status.
+	 * @param WP_Post $post       Post object.
+	 */
+	public static function maybe_clear_count_transients( $new_status, $old_status, $post ) {
+		global $wpdb;
+
+		/**
+		 * Get supported post types for count caching
+		 *
+		 * @since 1.26.2
+		 *
+		 * @param string  $new_status New post status.
+		 * @param string  $old_status Old post status.
+		 * @param WP_Post $post       Post object.
+		 */
+		$post_types = apply_filters( 'job_manager_count_cache_supported_post_types', array( 'job_listing' ), $new_status, $old_status, $post );
```

Comment:
As of this release, it seems like the general direction is using the `wpjm_` prefix for all new hooks. (See: #1034)